ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -331,10 +331,10 @@ StatusDialog {
 
                             if(root.swapAdaptor.validSwapProposalReceived) {
                                 if(root.swapAdaptor.swapOutputData.approvalNeeded) {
+                                    let approvalGasFeesFiat = root.swapAdaptor.currencyStore.getFiatValue(root.swapAdaptor.swapOutputData.approvalGasFees, Constants.ethToken)
                                     return root.swapAdaptor.currencyStore.formatCurrencyAmount(
-                                                root.swapAdaptor.swapOutputData.approvalGasFees,
+                                                approvalGasFeesFiat,
```

Comment:
Hmm now I'm wondering if we need to show the total fees here (for approval+swap) or just the approval ones until we're able to perform the swap...

Reply:
hmm thats a  good point, I'll ask Alisher

Reply:
https://github.com/status-im/status-desktop/issues/15439 created an issue to track this